pyproject.toml

```diff
@@ -51,7 +51,7 @@ test = [
 matter-server = "matter_server.server.__main__:main"
 
 [tool.codespell]
-ignore-words-list = "pase,"
+ignore-words-list = "pase,nd"
```

Comment:
Where is `nd` used?

Reply:
in the readme apparently (line 57)

Reply:
I, it's short for `Neighbor Discovery Protocol`. Maybe we should explain that by putting it in parenthesis after the full name the first time. Just a side note.

Reply:
adjusted it